app/users.php

```diff
@@ -1,87 +1,238 @@
-<?php require 'init.php';?>
+<?php require 'init.php'; ?>
+<?php include_once $_ENV['BASE_DIRECTORY'] . '/web-assets/app_nav/footer.php'; ?>
```

Comment:
Convert the HTML to twig template, and render using the basic layout, and you won't need to include header/footer.